_includes/manuals/2.1/3.2_foreman_installer.md

```diff
@@ -10,7 +10,7 @@ By default it will configure:
 * Apache HTTP with SSL (using a Puppet-signed certificate)
 * Foreman running under Puma (using systemd socket activation)
 * Smart Proxy with SSL (using a Puppet-signed certificate) configured for Puppet and TFTP
-* Puppetserver
+* Puppet server
```

Comment:
I still find it difficult to follow. First it was Puppet master, Puppetserver and now https://puppet.com/docs/puppet/6.18/server.html refers to Puppet Server. I think here we should follow the capitalization
```suggestion
* Puppet Server
```